templates/Segment_Image_Data.ipynb

```diff
@@ -188,9 +188,16 @@
    "outputs": [],
    "source": [
     "# generate and save deepcell input tifs\n",
+    "# set img_sub_folder param to None if tiff_dir does not contain one for fov data\n",
```

Comment:
I would change this to:
"set img_sub_folder param to None if the image files in tiff_dir are not in a separate sub folder "